pkgs/utf8proc.sh

```diff
@@ -0,0 +1,59 @@
+# ================================================================
+# Compile utf8proc
+# ================================================================
+
+[ -e $STAGE/utf8proc ] && ( set -xe
+    cd $SCRATCH
+
+    . "$ROOT_DIR/pkgs/utils/git/version.sh" JuliaStrings/utf8proc,v
+    until git clone --depth 1 --single-branch -b "$GIT_TAG" "$GIT_REPO"; do echo 'Retrying'; done
+    cd utf8proc
+    
+    # ------------------------------------------------------------
+
+    . "$ROOT_DIR/pkgs/utils/fpm/pre_build.sh"
+
+    (
+        case "$DISTRO_ID" in
+        'centos' | 'fedora' | 'rhel')
+            set +xe
+            . scl_source enable devtoolset-8
+            set -xe
+            export CC="gcc" CXX="g++"
+            ;;
+        'ubuntu')
+            export CC="gcc-8" CXX="g++-8"
+            ;;
+        esac
+
+        . "$ROOT_DIR/pkgs/utils/fpm/toolchain.sh"
+
+        mkdir -p build
+        cd $_
+
+        cmake                                       \
+            -DBUILD_SHARED_LIBS=ON                  \
+            -DCMAKE_BUILD_TYPE=Release              \
+            -DCMAKE_C_COMPILER="$CC"                \
+            -DCMAKE_CXX_COMPILER="$CXX"             \
+            -DCMAKE_C{,XX}_COMPILER_LAUNCHER=ccache \
+            -DCMAKE_C{,XX}_FLAGS="-fdebug-prefix-map='$SCRATCH'='$INSTALL_PREFIX/src' -g"   \
+            -DCMAKE_INSTALL_PREFIX="$INSTALL_ABS"   \
+            -G"Ninja"                               \
+            ..
+
+        time cmake --build .
+        # One test randomly failed on Ubuntu recently (Sep 28, 2019).
```

Comment:
I believe this is copied from other file and unrelated

Reply:
right. will remove it